vault/resource_database_secret_backend_role_test.go

```diff
@@ -38,6 +38,12 @@ func TestAccDatabaseSecretBackendRole_import(t *testing.T) {
 					resource.TestCheckResourceAttr("vault_database_secret_backend_role.test", "default_ttl", "3600"),
 					resource.TestCheckResourceAttr("vault_database_secret_backend_role.test", "max_ttl", "7200"),
 					resource.TestCheckResourceAttr("vault_database_secret_backend_role.test", "creation_statements.0", "SELECT 1;"),
+					resource.TestCheckResourceAttr("vault_database_secret_backend_role.role", "credential_config.ca_cert", "cert"),
+					resource.TestCheckResourceAttr("vault_database_secret_backend_role.role", "credential_config.ca_private_key", "key"),
+					resource.TestCheckResourceAttr("vault_database_secret_backend_role.role", "credential_config.key_type", "rsa"),
+					resource.TestCheckResourceAttr("vault_database_secret_backend_role.role", "credential_config.key_bits", "2048"),
+					resource.TestCheckResourceAttr("vault_database_secret_backend_role.role", "credential_config.signature_bits", "256"),
+					resource.TestCheckResourceAttr("vault_database_secret_backend_role.role", "credential_config.common_name_template", "{{.DisplayName}}_{{.RoleName}}"),
 				),
 			},
 			{
```

Comment:
I think `key_bits` and `signature_bits` are not returned by the API. If that is correct, then we will need to add them as ignore fields. Example:

https://github.com/hashicorp/terraform-provider-vault/blob/eba5e722e8782590c46f3f1a51d464179d7dbcbd/vault/resource_azure_secret_backend_test.go#L146

Reply:
I believe that is why the test is failing in CI

Reply:
+1 on the above, this should fix the failing import test